command/agent/local.go

```diff
@@ -160,6 +198,40 @@ func (l *localState) Services() map[string]*structs.NodeService {
 	return services
 }
 
+// SetCheckToken is used to configure an ACL token for a specific
+// health check. The token is used during check registration operations.
+func (l *localState) SetCheckToken(id, token string) {
```

Comment:
@ryanuber Can we roll this into the SetCheck? and also RemoveCheckToken into RemoveCheck?